src/operators/rasterscaling.md

```diff
@@ -0,0 +1,77 @@
+# RasterScaling
+
+The raster scaling operator scales/unscales the values of a raster by a given scale factor and offset.
+This allows to shrink and expand the value range of the pixel values needed to store a raster. It also allows to shift values to all-positive values and back. Keep in mind that scaling reduces the precision of the pixel values. We use the [GDAL](https://gdal.org/index.html) terms of [scale](https://gdal.org/programs/gdal_translate.html#cmdoption-gdal_translate-scale) and [unscale](https://gdal.org/programs/gdal_translate.html#cmdoption-gdal_translate-unscale). Scaling is often used to reduce memory consumption of rasters. Unscaling calculates the "real" value of each stored pixel.
```

Comment:
> Keep in mind that scaling reduces the precision of the pixel values. 

not always right? => "might reduce"

Reply:
> Scaling is often used to reduce memory consumption of rasters.

does only make sense when also converting the data type right? Or do you mean that data is often stored in a scaled fashion to save space and has to unscaled to get the real value?

Reply:
i reworked the docs in this commit: [aaa9676](https://github.com/geo-engine/docs/pull/33/commits/aaa967690250857a15aecf0b2993d2172004f14f)